examples/minimalist.py

```diff
@@ -53,24 +51,18 @@ def set_value(request):
         return _VALUES.get(key)
 
 
-# Create a service to serve our OpenAPI spec
-swagger = Service(name='OpenAPI',
-                  path='/__api__',
-                  description="OpenAPI documentation")
-
-
-@swagger.get()
-def openAPI_spec(request):
-    doc = CorniceSwagger(get_services())
-    my_spec = doc.generate('MyAPI', '1.0.0')
-    return my_spec
```

Comment:
Maybe move this to another example?

Reply:
I think the minimalist example should show the easiest way to archieve that?
How to generate the docs is mentioned above so I'm not sure if it makes sense to duplicate the example just show that.